testflinger_agent/agent.py

```diff
@@ -32,6 +32,11 @@ def __init__(self, client):
         self.set_state("waiting")
         self.advertised_queues = self.client.config.get("advertised_queues")
```

Comment:
we were ok with this previously because we weren't sending it to somewhere else, but in order to be consistent with types, let's make sure that this is a dict, even if it's empty:
`self.client.config.get("advertised_queues", {})`

Reply:
Makes sense.